bed/bedpe/modify.go

```diff
@@ -0,0 +1,21 @@
+package bedpe
+
+import "github.com/vertgenlab/gonomics/bed"
+
+//contactToMidpoint returns a bedpe where the contact has been collapsed to its midpoint
+func contactToMidpoint(bp BedPe) BedPe {
```

Comment:
Do we need to allocate memory and make new ones, or can we just overwrite the values?  Form a quick glance at the code that calls this function, it doesn't seem like we go back to the original.